patternfly/patternfly-elements
# fix: Allowing CTA link (light DOM) to inherit the cascaded font family

## pfe-cta

Per typography issue specs, CTA should inherit font-family from page instead of declaring full font-stack https://github.com/patternfly/patternfly-elements/issues/900#issuecomment-636883169


### Testing instructions



1. Go to CTA demo page: https://deploy-preview-1120--happy-galileo-ea79c4.netlify.app/elements/pfe-cta/demo/
2. add the following to a local stylesheet for the page
```
body {
    font-family: "Comic Sans MS";
}
```
3. Ensure the CTA inherits the awesome font

#### Browser requirements

Your component should work in all of the following environments:



### Ready-for-merge Checklist

Check off items as they are completed.  Feel free to delete items if they are not applicable.



### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
